app/views/content/train-to-be-a-teacher/teacher-training-interview.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Preparing for a teacher training interview
+subcategory: Your teacher training application
+description: |-
+  Prepare for a teacher training interview. Learn more about what the interview involves, what you might be asked, and how to get help to prepare.
+related_content:
+    Find out about subject knowledge enhancement : "/train-to-be-a-teacher/subject-knowledge-enhancement"
+    What teacher training is like : "/train-to-be-a-teacher/inital-teacher-training"
+date: "2023-06-09"
+keywords:
+  - adviser
+  - advisor
+  - ITT
+  - Initial Teacher Training
+  - QTS
+  - Qualified Teacher Status
+  - Teacher Training Adviser
+  - application
+  - applying for teacher training
+  - process
+  - interviews
+promo_content:
+    - content/train-to-be-a-teacher/promos/adviser-promo-interviews
+navigation: 20.30
+navigation_title: Preparing for interviews
+navigation_description: Find out how to prepare for a teacher training interview, with tips on what's involved, common questions, and to how to get help.
+---
+
+Congratulations on being invited for an interview. This is an important stage in successfully getting a place on a teacher training course.
+
+These tips can you prepare for a successful interview. The exact process will vary depending on the course provider.
+
+## Confirm your interview date
+
+The teacher training provider will email you with an interview date. If you can't make it then you may be able to ask for an alternative, but some providers may only offer interviews on specific days.
+
+Once the date is agreed you should recieve a confirmation email from the provider. Check this carefully to make sure you know exactly what to expect. For example, most providers ask for some type of presentation. Others may require maths or English tests to be completed in advance.
+
+## Prepare what you need to know
+
+It's a good idea to check the course provider's website to know what to expect. You can refer to this information at your interview.
+
+Consider some possible safeguarding scenarios and how you might address them. Make sure you've read the [statutory guidance for schools and colleges on safeguarding children](https://www.gov.uk/government/publications/keeping-children-safe-in-education--2).
+
+Also, take some time to find out about [the teachers’ standards](https://www.gov.uk/government/publications/teachers-standards) and [the national curriculum](https://www.gov.uk/government/collections/national-curriculum).
```

Comment:
I don't think we normally include 'the' in links to the teachers' standard or the national curriculum